### PR TITLE
Add @GitHubTeacher Bio

### PR DESCRIPTION
**Pull request by githubteacher**
This Pull Request adds a bio for @githubteacher as requested in #2.

I would :heart: some feedback on this bio @githubstudent and @nathos and @brntbeer.  If you have a few minutes, please take a :eyes: at these changes and let me know your :thought_balloon:s.

### CODE REVIEW

**Comment by nathos** [comment]
@githubteacher I'd :heart: to know what your favorite color is. Could we add that to your bio?

File: docs/githubteacher-bio.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# GitHub Teacher
+
+- **Name:** GitHub Teacher
+- **Occupation:** Teacher
+- **Hometown:** San Francisco
```
Comment (githubteacher) [inline docs/githubteacher-bio.md]: I think we should be more specific.  There are a lot of San Franciscos in the :earth_africa:.

Comment (iamdrmonkey) [inline docs/githubteacher-bio.md]: Below this, you should specify your favorite movie.

**Comment by nathos** [comment]
I think this looks great!

:shipit:

**Pull request merged**